packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/no-transform-sync-component-await-cookies.output.js

```diff
@@ -0,0 +1,8 @@
+import { cookies } from "next/headers";
+
+async function MyComponent() {
+  function asyncFunction() {
+    callSomething(/* TODO: await this async call and propagate the async correctly */
```

Comment:
Why can't we propagate the `async` here?

Reply:
This is just a simple example, it could be nested for many layers where hard to propagate. To make the propagation ideal, will take a lot of work. It's bit out of scope for this codemod

Reply:
Once you make the propagation work for one layer, you make it work for all layers. Only module boundaries are an issue.